Sources/SQL/Serialization/SQLSerializer+DataSubquery.swift

```diff
@@ -0,0 +1,8 @@
+extension SQLSerializer {
+    /// See `SQLSerializer`.
+    public func serialize(column: DataSubqueryColumn) -> String {
+        let query = serialize(query: column.query)
+        let serialized = "(\(query))"
```

Comment:
Super-nit: I've heard that string interpolation can be much slower than concatenation (see https://mjtsai.com/blog/2018/03/09/faster-swift-string-concatenation/). Would it be worth it to change to `"(" + query + ")"` (depends on how often this going to be called, of course)?